sagemaker-python-sdk/pytorch_mnist/pytorch_mnist_elastic_inference.ipynb

```diff
@@ -2,9 +2,7 @@
  "cells": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #29.            "https://sagemaker-sample-files.s3.amazonaws.com/datasets/image/MNIST/"
</pre><p>Will this if/else work? If the data is corrupted, then give it the same S3 path? Or do we need to provide a different backup mirror?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3159/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
<p>The if/else is more for different versions of PyTorch and that we account for all of them. They have different class variables. If the data is corrupted, the notebook will fail (as we wouldn't be able to download the data anyways) which will be picked up in the CI. In this case, we will have to manually investigate our data in sagemaker-example-files and replace it if necessary.</p><p> </p><p>But IMO, the chances of corruption is very unlikely as we are just reading the MNIST dataset most of the time. So, I don't foresee this being a huge problem</p>